docs/victorialogs/vlagent.md

```diff
@@ -270,6 +336,32 @@ See the docs at https://docs.victoriametrics.com/victorialogs/vlagent/ .
         TenantID for logs ingested via the Journald endpoint. See https://docs.victoriametrics.com/victorialogs/data-ingestion/journald/#multitenancy (default "0:0")
   -journald.timeField string
         Field to use as a log timestamp for logs ingested via journald protocol. See https://docs.victoriametrics.com/victorialogs/data-ingestion/journald/#time-field (default "__REALTIME_TIMESTAMP")
+  -kubernetes
```

Comment:
why prefix for log collection flags is `kubernetes`? None of them seems to be kubernetes specific, all of them seem to be generic for any logs collection kind

Reply:
Agree. What do you think about the `kubernetesCollector` prefix?